plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/local-pickup-select/index.tsx

```diff
@@ -47,7 +47,11 @@ export const LocalPickupSelect = ( {
 				highlightChecked={ true }
 				selected={ selectedOption }
 				options={ pickupLocations.map( ( location ) =>
-					renderPickupLocation( location, packageCount )
+					renderPickupLocation(
+						location,
+						packageCount,
+						selectedOption
```

Comment:
Day after feedback: instead of passing `selectedOption` here, it would be more fitting to pass `isSelected` instead. It's the parent component that decides which option is selected (hence having access to `selectedOption`), having all related logic to this aspect in the parent makes sense as well.